pkgs/by-name/le/letsdns/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+let
+  version = "1.2.1";
+in
+python3Packages.buildPythonApplication {
+  pname = "letsdns";
+  inherit version;
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "LetsDNS";
+    repo = "letsdns";
+    tag = version;
+    hash = "sha256-TwGVm7sEOPvUqtvaAuIU/X5W3H4VAC8dskNunt8UO0I=";
+  };
+
+  build-system = [
+    python3Packages.setuptools
+  ];
+
+  nativeCheckInputs = [
+    python3Packages.pytestCheckHook
+    versionCheckHook
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
```

```suggestion
  dependencies = with python3Packages; [
```
use of propogatedBuildInputs is discouraged

---

Fixed.